install/kubernetes/cilium/templates/cilium-preflight/deployment.yaml

```diff
@@ -22,16 +22,6 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
-      affinity:
-        podAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-          - labelSelector:
-              matchExpressions:
-              - key: "k8s-app"
-                operator: In
-                values:
-                - cilium
-            topologyKey: "kubernetes.io/hostname"
```

Comment:
Where did this default go?

Reply:
I miss this in `values.yaml`. Now, `affinity` is configurable as `.Values.preflight.affinity`